src/pages/docs/words.mdx

```diff
@@ -1,3 +1,190 @@
 import React from 'react';
+import Link from 'next/link';
+import { Tab, Tabs } from 'nextra-theme-docs';
+import { useState } from 'react'
 
-# Words
\ No newline at end of file
+### Publicly Access the words API Routes
+
+```js {1}
+[ Base URL: igboapi.com/api/v1/ ]
+
+```
+
+***
+
+###### Words Route
+
+```js copy
+GET /words
+```
+Get words in Dictionary
+
+ **Parameters**
+
+| Name   | Description |
+| :----- | :----: |
+| **keyword** *required* | Keyword for querying words (string) |
+| **page** | Page for results (integer) |
+| **range** | Page for result using [x,y] syntax, max range includes 25 documents (string) |
+| **strict**  | Searches for matching words that start and end with the provided keyword's letters (boolean) |
+| **dialects** | Includes dialect information in returned document (boolean) |
+| **examples** | Includes examples associated with returned document (boolean) |
+| **isStandardIgbo**  | Returns words that are marked as Standard Igbo (boolean) |
+| **pronunciation** | Returns words that have audio pronunciations (boolean) |
+| **nsibidi** | Returns words that have Nsịbịdị equivalents (boolean) |
+
+
+ **Responses**
+
+ The response for the route will be a plain JSON object similar to this:
+
+ ```
+ [
+    {
+        "wordClass": "string",
+        "definitions": [
+            "hunger; desire; eagerness"
+        ],
+        "examples": [],
+        "phrases": {
+            "(agụū) -gụ": {
+                "definitions": [
+                    "be hungry"
+                ],
+                "examples": []
+            },
+            "agụū mmīli": {
+                "definitions": [
+                    "thirst"
+                ],
+                "examples": []
+            },
+            "-gụ agụū": {
+                "definitions": [
+                    "hunger; desire; long for ( -gụ 2. desire)"
+                ],
+                "examples": []
+            }
+        }
+    }
+]
+ ```
```

Comment:
please use the accurate data response for this example since this isn't what's returned by the API